<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineSrc" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>10</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>11</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Папка spoi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>37</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>output (*.msv)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineDst" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>40</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bOpenSrc" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>выбрать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bOpenDst" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>40</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>выбрать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bStart" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>старт</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressConvertion" >
   <property name="geometry" >
    <rect>
     <x>17</x>
     <y>70</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value" >
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
